tensorflow/core/kernels/mkl/mkl_conv_ops.cc

```diff
@@ -77,20 +78,23 @@ struct MklConvFwdParams {
   MklConvFwdParams(memory::dims src_dims, memory::dims filter_dims,
                    memory::dims bias_dims, memory::dims dst_dims,
                    memory::dims strides, memory::dims dilations,
-                   memory::dims padding_left, memory::dims padding_right)
+                   memory::dims padding_left, memory::dims padding_right,
+                   MKL_TENSOR_FORMAT tf_fmt)
       : src_dims(src_dims),
         filter_dims(filter_dims),
         bias_dims(bias_dims),
         dst_dims(dst_dims),
         strides(strides),
         dilations(dilations),
         padding_left(padding_left),
-        padding_right(padding_right) {}
+        padding_right(padding_right),
+        tf_fmt(tf_fmt) {}
 };
 
 // With quantization, input, filter, and output can have different types
 // so we use different template parameter for each type
-template <typename Tinput, typename Tfilter, typename Tbias, typename Toutput>
+template <typename Tinput, typename Tfilter, typename Tbias, typename Toutput,
+          bool native_format>
```

Comment:
Same comment as in #42072, I don't think `native_format` needs to be a template parameter here. Can we make it a normal parameter?

Reply:
Done.